### ISSUE TITLE
Use of submenus

### ISSUE THREAD

**Issue by trashutz** [swingconnection-leipzig/swingconnection#296]
Just some thoughts...

The current submenu looks like that:
![grafik](https://github.com/swingconnection-leipzig/swingconnection/assets/84323290/61c80865-9ca0-4a24-8fa5-c58994cd6c3f)

I'm wondering if it's better to reduce the number of entries to make it simplier for new people to find relevant information.

E.g. "Tag der offenen Tür" and "Mattstedt" are only relevant for a short time each year. Maybe we can remove them if they are not relevant at the time.

For "Fridey Night Balboa", "Swing Brunch" and "Swing Tea Time" I would like to have specific dates on the pages and no placeholders like `xx.xx.2024` or `tba`. But of course this is not possible if there are no fixed dates.

"Swing Tea Time" will be coming for the summer program. But "Swing Brunch" not.. Maybe we can also remove it for now until it will happen again more regularly?

Another thing is "Minor Swing". The name alone doesn't say anything about social. It's in the events menu, but maybe it's better to more explicit like "Minor Swing Social" or something like that.

**Comment by hnolzen** [comment]
> I'm wondering if it's better to reduce the number of entries to make it simplier for new people to find relevant information.

In general a good idea. In my opinion we can remove some entries from the submenu until the events take place (regularly) again. However, I also think they also have a bit of an advertising function. New people can see what events the SCL generally offers. Otherwise, only Minor Swing and FNB would remain there.

Of course we have to bring the individual pages up to date and ideally add pictures. And the events have to be reactivated.

> Another thing is "Minor Swing". The name alone doesn't say anything about social. It's in the events menu, but maybe it's better to more explicit like "Minor Swing Social" or something like that.

Good idea! Let's rename it.

**Comment by hnolzen** [comment]
> For "Fridey Night Balboa", "Swing Brunch" and "Swing Tea Time" I would like to have specific dates on the pages and no placeholders like xx.xx.2024 or tba. But of course this is not possible if there are no fixed dates.

You are right. This is not ideal. I will try to reformulate it, so that when the event is inactive we don't use these placeholders but something like "We will announce the next date here soon.".

**Comment by trashutz** [comment]
> In my opinion we can remove some entries from the submenu until the events take place (regularly) again. However, I also think they also have a bit of an advertising function. New people can see what events the SCL generally offers. Otherwise, only Minor Swing and FNB would remain there.

Yeah, I think we need a define a compromise. Having pages for events that didn't happen for over a year and no plan for a new date, is IMO just a bit sad and can be removed. And as long as the events don't take place we don't need advertising for them.

I would suggest we find maybe a way to promote events for members over other communication channels and the website is mainly for external promotion.

**Comment by hnolzen** [comment]
What do you think about removing Swing Brunch and Swing Tea Time until a new date is set? 

As for Mattstedt, could we rename it to "Mattstedt 2025" and keep it? 

I think an Open Door will happen within the next months. However I am not sure about the date. I think it will be announced when the summer programme is published.

> I would suggest we find maybe a way to promote events for members over other communication channels and the website is mainly for external promotion.

We have Telegram, FB, Insta as other advertising channels. But I know of SCL members who visit the website to check for event announcements. As long as the website is up to date, it's also a good way for people who don't use social media that much. Maybe we should also set up an email newsletter in the future!

**Comment by trashutz** [comment]
> What do you think about removing Swing Brunch and Swing Tea Time until a new date is set?

I'll organize two tea dance socials for the summer program. 18.8 and 22.9 should be the dates. But I think they are not official yet (https://cloud.wechange.de/apps/onlyoffice/12697075).

> As for Mattstedt, could we rename it to "Mattstedt 2025" and keep it?

If you want to. But then I would like to update the text. Because having the description from this year until there is new information sometime next year is not great.

> I think an Open Door will happen within the next months. However I am not sure about the date. I think it will be announced when the summer programme is published.

Current summer program says 29.9.

> We have Telegram, FB, Insta as other advertising channels. But I know of SCL members who visit the website to check for event announcements. As long as the website is up to date, it's also a good way for people who don't use social media that much. 

Yeah, you're right. I think what I wanted to say is, that we shouldn't promote events that wont happen in the near future. And promotion I meant is to promote ideas to SCL members who want to organize events.

> Maybe we should also set up an email newsletter in the future!

I support the ideo of a newsletter. Question is why will take the time to set it up and write them...

**Comment by hnolzen** [comment]
> I'll organize two tea dance socials for the summer program. 18.8 and 22.9 should be the dates. 

Great! Maybe then we can keep the submenu for the Tea Time?!

> Because having the description from this year until there is new information sometime next year is not great.

You are right. Maybe we can remove it from the menu and add it again when we know the exact date.

> Current summer program says 29.9.

Also great! Maybe then we can also keep the submenu for the Tea Time?!

**Comment by hnolzen** [comment]
> I support the ideo of a newsletter. Question is why will take the time to set it up and write them...

This is probably a project of the new board or could be a task of one of the new advisory board members?!
